packages/aws_bedrock/data_stream/invocation/elasticsearch/ingest_pipeline/default.yml

```
@@ -284,6 +284,38 @@ processors:
       field: gen_ai.response.id
       copy_from: aws_bedrock.invocation.output.output_body_json.id
       ignore_failure: true
+  - script:
+      tag: append_input_message_content_kinds
+      if: ctx.aws_bedrock?.invocation?.input?.input_body_json?.messages != null && ctx.aws_bedrock.invocation.input.input_body_json.messages instanceof List
+      lang: painless
+      source: |
+        if (ctx.aws_bedrock.invocation.input.messages_content_kinds == null) {
+            ctx.aws_bedrock.invocation.input.messages_content_kinds = [];
+        }
+        // using a Set to avoid duplicates
+        def content_kinds_set = new HashSet(ctx.aws_bedrock.invocation.input.messages_content_kinds);
```

```suggestion
        // using a Set to avoid duplicates
        def content_kinds_set = new HashSet(ctx.aws_bedrock.invocation.input?.messages_content_kinds ?: []);
```

`?:` is the [Elvis operator](https://www.elastic.co/guide/en/elasticsearch/painless/current/painless-operators-reference.html#elvis-operator).